src/modules/[chain]/block/[height].vue

```diff
@@ -38,7 +38,7 @@ const remainingBlocks = computed(() => {
 })
 
 const estimateTime = computed(() => {
-  const seconds = remainingBlocks.value * Number((store.blocktime / 1000).toFixed()) * 1000
+  const seconds = remainingBlocks.value * store.blocktime
```

Comment:
seems it can be "1.53432424242" in some cases. 
So will be good to have a round on the final result. 
`const seconds = Number((remainingBlocks.value * store.blocktime).toFixed(2))`

Reply:
Right, 2 decimal places will be enough.